File: 1_tow_sum.go
package main

import "fmt"

func main() {
	nums := []int{3, 2, 3}
	t := 6
	var result = twoSum(nums, t)
	fmt.Println(result)
}

func twoSum(nums []int, target int) []int {
    first := 0
	for first < len(nums) - 1 {
		for second := first + 1; second < len(nums); second++{
			if nums[first] + nums[second] == target{
				return []int{first, second}
			}
		}
		first++
	}
	return nil
}

File: 1_tow_sum.py
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        first: int = 0
        while first < len(nums) - 1:
            for second in range(first + 1, len(nums)):
                if nums[first] + nums[second] == target:
                    return [first, second]
            first += 1


if __name__ == '__main__':
    result = Solution()
    print(result.twoSum([3, 2, 3], 6))
